### DISCUSSION TITLE
How do I change units in TukTuk?

### DISCUSSION THREAD

**Discussion by toastygm** [bareboat-necessities/lysmarine_gen#347] [Q&A]
I'm trying to use TukTuk navigation, but the units are SI units, and I want different ones.

Depth is in meters, I want feet.
TWS is in meters, I want knots.
AWS in in meters, I want knots.

I'm not sure whether to configure this in TukTuk, or whether I need to configure it in Signalk.  I have no idea how to change the units in Signalk (and I don't know if that is even a good idea), or how to change the units in TukTuk.

**Comment by mgrouch** [comment]
TukTuk is a bit rusty and seems no longer actively maintained. Can you use AvNav instead?
